presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoRelations.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.plugin.geospatial;
+
+import com.facebook.presto.operator.scalar.AbstractTestFunctions;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.BooleanType.BOOLEAN;
+
+public class TestGeoRelations
+        extends AbstractTestFunctions
+{
+    @BeforeClass
+    protected void registerFunctions()
+    {
+        GeoPlugin plugin = new GeoPlugin();
+        registerTypes(plugin);
+        registerFunctions(plugin);
+    }
+
+    @Test
+    public void testStContains()
+    {
+        assertFunction("ST_Contains(ST_GeometryFromText(null), ST_GeometryFromText('POINT (25 25)'))", BOOLEAN, null);
```

Comment:
Consider introducing helper function assertContains(wkt, otherWork, expectedResult) to simplify these tests:

```
assertContains("POINT (20 20)", "POINT (25 25)", false);
assertContains("MULTIPOINT (20 20, 25 25)", "POINT (25 25)", false);
...
```